### PR TITLE
aad filter sample

### PR DESCRIPTION
**Pull request by yaweiw**

### CODE REVIEW

**Comment by yaweiw** [comment]
@yungez @xscript @aovechki @ZhijunZhao  please review

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/pom.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>angularjs</artifactId>
+            <version>1.4.3</version>
```
Comment (yungez) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/pom.xml]: put this as property to align styling with your plugin?

Comment (yaweiw) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/pom.xml]: this dependency will be removed. not needed at all.

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+## Overview
+This sample illustrates how to use `azure-ad-integration-spring-boot-autoconfigure package` to plugin JWT token filter into Spring Security filter chain. The filter injects `UserPrincipal` object that is associated with the thread of the current user request. User's AAD membership info, along with token claimsset, JWS object etc. are accessible from the object which can be used for role based authorization. Methods like `isMemberOf` is also supported.
```
Comment (yungez) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/README.md]: ` before package?

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            },
+            $httpProvider
+        );
+
+```
+
+### Give it a run
+* Go to `path-to-azure-spring-boot-starters`, run `mvn clean package`.
+* `cd activedirectory\azure-ad-integration-spring-boot-autoconfigure`
+* `mvn install`
```
Comment (ZhijunZhao) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure-sample/README.md]: if you do 'mvn package' in the root folder, i think it also runs without the mvn install

**Pull request merged**